### ISSUE TITLE
Chatbot text scrolls through title text

### ISSUE THREAD

**Issue by itsnotmik** [DrAlzahraniProjects/csusb_fall2024_cse6550_team1#58]
Can be viewed below. Text is currently flowing over and showing behind the title message

![image](https://github.com/user-attachments/assets/100c5f7e-803a-4ba9-a886-9314b2622d2c)

**Comment by Jagadeesh-ajjada** [comment]
Hi @tejaswi-142 once you done with this, update the task status to avoid duplicate work contribution.

**Comment by tejaswi-142** [comment]
I've created a branch called issue-58-fix. Please use this @Adenkri45 
lets merge the code in this branch first so that we can avoid merge conflicts in main.

**Comment by Adenkri45** [comment]
Sure will work on that.

**Comment by vamsikrishnabandaru3** [comment]
I have worked on the issue. @tejaswi-142 if you are still working on the code, can we test the code I have untill you worked on it.

**Comment by vamsikrishnabandaru3** [comment]
please let me know so, that I can create a pull request.

**Comment by Jagadeesh-ajjada** [comment]
If possible, the both of you should set up a Zoom call to merge your code.

**Comment by tejaswi-142** [comment]
While @Adenkri45 and I were fixing this issue, we faced a blocker where it doesn't work for dark mode and light mode both.
- So, this is the solution I came up with. 
- I've added a custom theme, now our custom theme colors are used in the web app no matter what the user's system theme is i.e. *dark* or *light* theme.

This custom theme will also give us more freedom while altering CSS in the future. We don't have think about dark and light themes.
